Sources/TensorFlow/LazyTensor/LazyTensorOperation.swift

```diff
@@ -105,6 +105,10 @@ class LazyTensor: _AnyTensorHandle {
         }
     }
 
+    static func onAllOperations(_ perform: (LazyTensorOperation) -> ()) {
```

Comment:
```suggestion
    static func forEachOperation(_ perform: (LazyTensorOperation) -> Void) {
```

1. It is more common for such methods to be called `forEachXXX`, based on the precedent [`forEach(_:)`](https://developer.apple.com/documentation/swift/array/1689783-foreach) in the Swift standard library.
2. It is more common (throughout the standard library) to write `-> Void` for void returns instead of `-> ()`.
3. Would you like to make this method `throws`-`rethrows`? This would make it easier to put things that throw inside the body closure and work nicely with XCTest.

Reply:
Please also rename `onLiveOperations` to `forEachLiveOperation`.

Reply:
Thanks! Done.